homeassistant/components/generic/strings.json

```diff
@@ -5,6 +5,12 @@
       "unknown": "[%key:common::config_flow::error::unknown%]",
       "already_exists": "A camera with these URL settings already exists.",
       "unable_still_load": "Unable to load valid image from still image URL (e.g. invalid host, URL or authentication failure).  Review log for more info.",
+      "unable_still_load_401": "Unable to load valid image from still image URL: Please check your username or password. (Unauthorized)",
```

Comment:
I suggest we try to give the user some helpful words here of what to do -- for example: The camera may require a username and password, or its not correct.

Or below for 404: The url is incorrect.  For a 5XX: The camera replied with a server error.

Would it be reasonable to combine 401 and 403 and 500 and 503? I think i'd suggest making three error code slike: unable_still_load_auth, unable_still_load_not_found, unable_still_load_server_error. I'm not sure if the user would do anything different if they got a 500 vs a 503.

Reply:
Thanks for the review!

The original request was to make the errors more specific, but I agree, not much to gain unless we can advise the user on a more specific action to take.

I've grouped as you suggested and put some more helpful wording in. The HTTP error code is still in the log message if the user wants more detail.